website/versioned_docs/version-22.x/TutorialReactNative.md

```diff
@@ -201,22 +201,3 @@ jest.mock('Text', () => {
 In other cases you may want to mock a native module that isn't a React component. The same technique can be applied. We recommend inspecting the native module's source code and logging the module when running a react native app on a real device and then modeling a manual mock after the real module.
 
 If you end up mocking the same modules over and over it is recommended to define these mocks in a separate file and add it to the list of `setupFiles`.
-
-### `@providesModule`
```

Comment:
Should probably keep this as it's the versioned docs, and only remove it form `docs/`

Reply:
Ah, it was removed in new docs in #7147.